### PR TITLE
Mention lego-bricks in main README.md

### PR DESCRIPTION
**Pull request by norbye**
# Description

Add stuff to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,16 @@
 
 ## Quick Start
 
+Before running LEGO-WEBAPP, you have to compile LEGO-BRICKS
+
+```bash
+$ cd packages/lego-bricks
```
Comment (ivarnakken) [inline README.md:22]: You can build from the ropt as well, no? I believe you build both webapp and bricks then

File: README.md [hunk 2]
```diff
@@ -16,6 +16,16 @@
 
 ## Quick Start
 
+Before running LEGO-WEBAPP, you have to compile LEGO-BRICKS
```
Comment (ivarnakken) [inline README.md:19]:
```suggestion
Before running LEGO-WEBAPP, you have to compile LEGO-BRICKS.
```
Perhaps we should also describe what lego-bricks actually is🧐

**Review by ivarnakken — COMMENTED** [review]
Very nice to have!

File: README.md (hunk 1 above)
Comment (norbye) [reply README.md:22]: Ahh that's probably true, will check tomorrow

File: README.md (hunk 2 above)
Comment (norbye) [reply README.md]: Updated with a little bit more of a description now

**Review by danielyanghansen — APPROVED** [review]
Looks good. For some reason, it's only necessary to manually build bricks once, and then it builds automatically later.
Don't know if that's necessary to mention

**Pull request merged**